src/Symfony/Component/Debug/README.md

```diff
@@ -31,7 +31,7 @@ Note that the `Debug::enable()` call also registers the debug class loader
 from the Symfony ClassLoader component when available.
```

Comment:
this should be updated too. The DebugClassLoader is in the Debug component since a long time (the ClassLoader one is deprecated)